spiceaidocs/docs/components/data-connectors/mssql.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Microsoft SQL Server Data Connector'
+sidebar_label: 'Microsoft SQL Server'
+description: 'Microsoft SQL Server Data Connector'
+---
+
+## Federated SQL query
```

Comment:
Be consistent with other data connectors and provide an introduction.

Reply:
Updated - thank you